operator/src/main/resources/postgres.yaml

```diff
@@ -0,0 +1,39 @@
+# PostgreSQL StatefulSet
```

Comment:
should we name this file `example-postgres` or `demo-postgres` ?

Reply:
👍